app/templates/_bower.json

```diff
@@ -36,24 +36,32 @@
         <%_ if (websocket == 'spring-websocket') { _%>
         "sockjs-client": "1.0.3",
         "stomp-websocket": "2.3.4",
+        <%_ } _%> <%_ if (enableSocialSignIn) { _%>
+        "bootstrap-social": "4.10.1",
         <%_ } _%>
+        "font-awesome": "4.4.0",
```

Comment:
I create button with string and css:
![capture d ecran 2015-10-19 a 10 45 24](https://cloud.githubusercontent.com/assets/4958079/10573215/8a8d0140-764e-11e5-8ccb-b249629fd25e.png)